src/map/entities/charentity.h

```diff
@@ -456,6 +456,7 @@ class CCharEntity : public CBattleEntity
     bool   isPacketFiltered(std::unique_ptr<CBasicPacket>& packet);
 
     bool pendingPositionUpdate;
+    bool m_SendServerStatus;
```

Comment:
I know our member variable naming is all over the place, all over the place, but can we have this as: `sendServerStatus_ = false;`, then you don't need to initialize in the constructor.